Makefile

```diff
@@ -45,7 +45,7 @@ generate-controller-manifests:
 	hack/update-jobset-controller-manifests.sh
 .PHONY: generate-controller-manifests
 
-generate: regen-crd generate-clients generate-controller-manifests
+generate: generate-clients regen-crd generate-controller-manifests
```

Comment:
Why do you change the order?

Reply:
just that having the clients updated first seemed more important to me ;)

Reply:
Makes sense :)